terraform/projects/app-ecs-instances/instance-user-data.tpl

```diff
@@ -32,10 +32,15 @@ else
   echo "[$(date '+%H:%M:%S %d-%m-%Y')] attach-volume: /dev/$DEVICE is already formatted: $(file -s /dev/$DEVICE)"
 fi
 
-# Allow prometheus container user access to read/write/execute within container
+# Allow prometheus container user access to read/write metrics to persistent volume mount from within a container
 mkdir -p /ecs/prometheus_data
+
+#Cretate prometheus & grant access to prometheus data files. We create the group & match it to the gid
```

Comment:
Would be good to be explicit that the prometheus container runs as nobody which has UID `65534` by default (with a link to prove that if easy to find)

Reply:
I don't know what you mean by link in this context. If you mean that this is the correct method I would say: 

'docker run prom/prometheus'

in a new terminal: 

docker exec -it <<hash of container>> /bin/sh
/prometheus $ whoami
nobody
/prometheus $ id
uid=65534(nobody) gid=65534(nogroup)

Through this you can verify the permissions. If this is not what you mean by link please clarify.